sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Features Added
 
+- Report metrics "CPU Total" and "Committed Memory".
```

Comment:
Just say report CPU and memory metrics.

@cijothomas Your thoughts?

Reply:
Can we make it precise and clear? This is one example of how to be precise and clear https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.Process#processmemoryvirtual